data/unaligned_labeled_mask_online_dataset.py

```diff
@@ -77,15 +83,17 @@ def sanitize(self):
         print('Sanitizing images and labels paths')
         print('--- DOMAIN A ---')
```

Comment:
We may want to use tqdm and have a progression bar here. In my test on 15k images dataset it takes a long time, so it'd be useful to see the progress.